addons/mail/tests/__init__.py

```diff
@@ -2,6 +2,7 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from . import test_mail_channel
+from . import test_mail_channel_as_guest
```

Comment:
Generally, we use a specific file naming for UI tests, like ``test_mail_channel_ui`` . Could be ``test_discuss_ui`` probably, if you want to include some discuss tours.